server/middleware/fileStorage.js

```diff
@@ -196,27 +196,36 @@ var addFilesForZip = function(){
     }
 }
 
-var zipFile = function(){
-    return function(req,res){
+var zipFile = function () {
+    return function (req, res) {
         console.log(req)
         var archive = archiver('zip');
-        archive.on('error',function(err) {
-            res.status(500).send({error: err.message});
+        archive.on('error', function (err) {
+            res.status(500).send({ error: err.message });
         });
-        res.on('close', function() {
+        res.on('close', function () {
             console.log('Archive wrote %d bytes', archive.pointer());
             return res.status(200).send('OK').end();
         });
-        var zipName = req.params.assignmentID+'.zip'
+        var zipName = req.params.assignmentID + '.zip'
         res.attachment(zipName);
         archive.pipe(res);
         req.files.forEach(file => {
-            archive.append(fs.createReadStream(file.path), {name: file.name})
+            archive.append(fs.createReadStream(file.path), { name: file.name })
         });
         archive.finalize();
     }
 }
-
+var createFile = function (req, res, next) {
+    fs.writeFile(process.cwd() + "/temp/file.txt", req.body.text, function (err) {
+        if (err) {
+            res.status(500).send({ error: err });
+        }
+        console.log("File Created");
+        res.status(200).send({ success: true, message: "File Created." });
```

Comment:
Instead of returning res, throw an error, or return an appropriate message. Also, please add an example of how to use this util.